### DISCUSSION TITLE
ObtainiumJet [abandoned]

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [ImranR98/Obtainium#624] [Show and tell]
Hello!

I love your app, it is so useful and convenient.
But I also hate flutter, so it was inevitable that I would do this one day.

I started an unofficial remake *(well duh, if it was official then you would be the one making it)* of Obtanium but with Jetpack Compose.
I decided to name it ObtainiumJet, if you don't like me using that name then I can change it.

If you are interested in learning kotlin and/or Jetpack Compose then feel free to contribute!
But alas this is not a discussion to ask for help, don't get me wrong.

I was just excited to start this remake and wanted to share it as soon as possible.
If you actually run the app, it won't have anything more than the navbar, since I only just started working on it.

[The repo is here](https://github.com/ArjixWasTaken/Obtainium-Jet)

**Comment by ImranR98** [comment]
Yeah, writing the app in Flutter was definitely not a smart move but it was my only option. Things like silent updates seem very difficult (impossible?) to do in Flutter (and it's not like we need the cross-platform support). So I'm definitely looking forward to seeing development on your version!
